<launch>

    <!-- Parameters -->
    <arg name="RecordPrefix" default="$(find map_update)/record/" />
    <arg name="RotatedRecordPrefix" default="$(find map_update)/rotatedrecord/" />
    <arg name="ReadMmapfile" default="arg1.pcd" />
    <arg name="SavePcdfile" default="voxCloudmap_2.pcd" />  <!--arg只是变量名-->

    <param name="RecordPrefix" type="string" value="$(arg RecordPrefix)" />
    <param name="RotatedRecordPrefix" type="string" value="$(arg RotatedRecordPrefix)" />
    <param name="ReadMmapfile" type="string" value="$(arg ReadMmapfile)" />
    <param name="SavePcdfile" type="string" value="$(arg SavePcdfile)" />

    <rosparam file="$(find map_update)/config/params_update.yaml" command="load" />
    

    <!--- Map Update -->
    <node name="map_update_node"  pkg="map_update" type="debug_map_update"     output="screen" 	respawn="false"/>

    <!--- Run Rviz-->
    <!--
    <node pkg="rviz" type="rviz" name="$(arg project)_rviz" args="-d $(find lio_sam)/launch/include/config/rviz.rviz" />
     -->

</launch>
